config/samples/atlas_v1_atlasproject.yaml

```diff
@@ -6,4 +6,10 @@ spec:
   name: Test Atlas Operator Project
   projectIpAccessList:
     - ipAddress: "192.0.2.15"
-      comment: "IP address for Application Server A"
\ No newline at end of file
+      comment: "IP address for Application Server A"
+  projectMaintenanceWindow:
+    dayOfWeek: 7
+    hourOfDay: 18
+    autoDefer: false
+    #startASAP: true
+    #defer: true
```

Comment:
Consider adding another example file (e.g. `atlas_v1_atlasproject_with_maintenance_window.yaml`) with this object. Also, please remove commented fields if they are not a part of the example.